src/main/java/codeu/model/data/Conversation.java

```diff
@@ -36,12 +37,13 @@
    * @param title the title of this Conversation
    * @param creation the creation time of this Conversation
    */
-  public Conversation(UUID id, UUID owner, String title, Instant creation) {
+  public Conversation(UUID id, UUID owner, String title, Instant creation, String conversationUser) {
     this.id = id;
     this.owner = owner;
     this.creation = creation;
     this.title = title;
     this.isPrivate = false;
+    this.conversationUser = conversationUser;
```

Comment:
I'm not sure I understand; what is `conversationUser`? What does it do and where is it set?

Reply:
To create a new conversation you can add users to it (kinda ties to the whole making conversations private), I made that "added User" the conversationUser in the constructor. I will change the name so its less confusing

Reply:
Done :)

Reply:
Oh, okay, I think I understand.
Why is it a part of the constructor?

Reply:
I was thinking of conversation users as an attribute of a conversation. I'm open to any suggestions for this, I want to change it from a String to a list of users so that more than one person can be added to a conversation.

Reply:
Ah, that makes more sense. I wouldn't put it on the constructor, though.

I'll accept the PR so you can keep working, but it'd be good to meet up to talk about this.